exposed-java-time/src/test/kotlin/org/jetbrains/exposed/DefaultsTest.kt

```diff
@@ -379,16 +374,7 @@ class DefaultsTest : DatabaseTestsBase() {
 
                 val actual = SchemaUtils.statementsRequiredToActualizeScheme(foo)
 
-                if (currentDialectTest is MysqlDialect) {
-                    // MySQL and MariaDB do not support CURRENT_DATE as default
-                    // so the column is created with a NULL marker, which correctly triggers 1 alter statement
-                    val tableName = foo.nameInDatabaseCase()
-                    val dateColumnName = foo.defaultDate.nameInDatabaseCase()
-                    val alter = "ALTER TABLE $tableName MODIFY COLUMN $dateColumnName DATE NULL"
-                    assertEquals(alter, actual.single())
-                } else {
-                    assertTrue(actual.isEmpty())
-                }
+                assertTrue(actual.isEmpty())
```

Comment:
In this test's table object, there are 2 `datetime()` columns that are exactly the same except for name. Could one be removed if it's not needed for anything?